### ISSUE TITLE
Add aria-labelledby aria-describedby attributes to our markup for fields

### ISSUE THREAD

**Issue by Shelob9** [CalderaWP/Caldera-Forms#575]
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-describedby_attribute

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

**Comment by Desertsnowman** [comment]
closed via ca3ac55

**Comment by Shelob9** [comment]
I am reopening this.

aria-describedy and aria-labeledby attributes should be on the form input, not the wrapping container. https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Forms/How_to_structure_an_HTML_form#The_aria-labelledby_attribute

Also we should only used describedby when there is a description. Don't use it and have it link to nothing.

**Comment by Desertsnowman** [comment]
@Shelob9 - we don't need the labelledby attribute since we do use `<label>` with a `for` attribute. In cases where labels are not shown, we can then use the `aria-label` .

**Comment by Desertsnowman** [comment]
closed via  c084c80 and a0974a2
